addons/portal/views/portal_templates.xml

```diff
@@ -389,93 +421,165 @@
     </template>
 
     <template id="portal_my_details">
+        <t t-call="portal.portal_layout"><div>
+            <t t-set="page_title" t-value="'Manage Addresses'"/>
+            <t t-set="additional_title" t-value="page_title"/>
+            <t t-set="show_side_bar" t-value="True"/>
+            <div class="col-12 col-xl order-xl-1 o_portal_details">
+                <div class="row">
+                    <div class="col-lg-12">
```

Comment:
same here (in the complete template) regarding a lot of imbricated div row/col. Could we use flex instead ?

Reply:
Or simply scrap it, as each col-12 will actually take a new line in the grid system ?